### DISCUSSION TITLE
課題10についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#30] [Q&A]
https://github.com/nsc-kiwamu/traning-takeda/pull/29#issue-1303097253

↑上記のリンク先にて質問を記載させていただいております。
よろしくお願いいたします。

**Comment by nsc-kiwamu** [comment]
https://github.com/nsc-kiwamu/traning-takeda/pull/29#issuecomment-1183223998
